### ISSUE TITLE
Feature Request : RSS Feed Compatibility 

### ISSUE THREAD

**Issue by kabitokaii** [nyaadevs/nyaa#425]
RSS Parsers like Sonarr find the feed to be incompatible. 

Nyaa.se had an API for auto-downloaders to search and parse daily. Something like this would be awesome.

**Comment by CounterPillow** [comment]
The [W3C validator says our RSS feed is valid](https://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Fnyaa.si%2F%3Fpage%3Drss), so the issue isn't with nyaa.si but with Sonarr. Try asking them.

**Comment by kabitokaii** [comment]
Let me rephrase my request to add Magnet links to the RSS feed. I figured
out that that's whats missing.

**Comment by TheAMM** [comment]
Use `&magnets` in your feed url. Many clients do not dig magnets, so we default to torrents.

**Comment by CounterPillow** [comment]
(Also, Sonarr should really support torrent files as `<link>`s, I don't see why it doesn't do that already)

**Comment by kabitokaii** [comment]
I think its that torrents are messy.

**Comment by ReimuHakurei** [comment]
The feed as is works fine in uTorrent and other torrent clients. Having
magnets breaks many of them. If you want magnets, add &magnets to the URL
as suggested above.
